docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -118,7 +118,8 @@ If you are using CommonJS, downgrade `node-fetch` to v2.
 
 ### WebSocket throttle [#createClientThrottle]
 
-By default, the client throttle the WebSocket messages sent to 100 milliseconds.
+By default, the client throttles the WebSocket messages sent to 100
+milliseconds.
```

Comment:
Just a minor typo in the documentation, unrelated to this change. Including it here, because I didn't feel like opening an entire PR for it.